Utils/src/PhotonIDisoProducer.cc

```diff
@@ -100,6 +107,22 @@ PhotonIDisoProducer::PhotonIDisoProducer(const edm::ParameterSet& iConfig):
   debug(iConfig.getUntrackedParameter<bool>("debug",true))
 {
 
+  effArEtaLow_           = iConfig.getParameter <std::vector<double>> ("effArEtaLow");
```

Comment:
all of these can be initialized in the constructor initialization list (like the input tags already are)